Goobi/src/de/sub/goobi/metadaten/Metadaten.java

```diff
@@ -3098,15 +3099,59 @@ boolean canCreate(MetadataGroupType type) {
      *             multi-select input
      */
     public List<RenderableMetadataGroup> getMyGroups() throws ConfigurationException {
-        List<MetadataGroup> records;
-        if (myDocStruct == null || (records = myDocStruct.getAllMetadataGroups()) == null) {
+        if (myDocStruct == null) {
             return Collections.emptyList();
         }
+
+        List<MetadataGroup> records = myDocStruct.getAllMetadataGroups();
+        if (records == null) {
+            records = Collections.emptyList();
+        }
+        List<MetadataGroupType> defaultGroupTypes = myDocStruct.getType().getAllDefaultDisplayMetadataGroups();
+        if (records.isEmpty() && defaultGroupTypes.isEmpty()) {
+            return Collections.emptyList();
```

Comment:
no need to create a new list instance, just return records or defaultGroupTypes.

Reply:
`records` is a `List<ugh.dl.MetadataGroup>`, `defaultGroupTypes` is a `List<ugh.dl.MetadataGroupType>`, but this method must return a `List<de.sub.goobi.metadaten.RenderableMetadataGroup>`, so everything should be right here. By the way, `Collections.emptyList()` does **not** create a new instance of a list but uses an unmodifiable constant.